spec/support/equipment_model_helper.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module EquipmentModelHelper
+  # traits are the private methods defined in this module
+  # a findable model with a category is created with
+  # traits: [:findable, [:with_category, cat: cat]]
+  def mock_eq_model(traits: [], **attrs)
+    model = instance_spy('EquipmentModel', **attrs)
+    traits.each { |trait, *args| send(trait, model, *args) }
+    model
+  end
+
+  def mock_eq_models(count: 1, traits: [], **attrs)
+    Array.new(count) { mock_eq_model(traits: traits, **attrs) }
+  end
+
+  def stub_collection_methods(models)
+    allow(EquipmentModel).to receive(:all).and_return(models)
+    allow(EquipmentModel).to receive(:where).and_return([])
```

Comment:
So this is explicitly breaking all scopes? I jumped down here to see where everything was coming from so it's possible I haven't quite found the place where this method is used for that purpose yet.